modules/cnf-topology-aware-lifecycle-manager-installation-cli.adoc

```diff
@@ -51,10 +51,10 @@ $ oc get csv -n openshift-operators
 ----
 +
 .Example output
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-NAME                                                      DISPLAY                            VERSION               REPLACES   PHASE
-topology-aware-lifecycle-manager.4.10.0-202206301927      Topology Aware Lifecycle Manager   4.10.0-202206301927              Succeeded
+NAME                                                             DISPLAY                            VERSION                      REPLACES              PHASE
+topology-aware-lifecycle-manager.{product-version}.0-xxxxxxxxxxxx     Topology Aware Lifecycle Manager   {product-version}.0-xxxxxxxxxxxxrelease                        Succeeded
```

Comment:
For 4.10, the csv version format is 4.10.0-xxxxxxxxxxxx
but for 4.11+, the version is changed to 4.11.x  (e.g., 4.11.0, 4.11.1)

Reply:
fixed